tikv/deploy-tikv-using-ansible.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Install and Deploy TiKV Using Ansible
+category: user guide
+---
+
+# Install and Deploy TiKV Using Ansible
+
+This guide describes how to install and deploy TiKV using Ansible. Ansible is an IT automation tool that can configure systems, deploy software, and orchestrate more advanced IT tasks such as continuous deployments or zero downtime rolling updates.
+
+[TiDB-Ansible](https://github.com/pingcap/tidb-ansible) is a TiDB cluster deployment tool developed by PingCAP, based on Ansible playbook. TiDB-Ansible enables you to quickly deploy a new TiKV cluster which includes PD, TiKV, and the cluster monitoring modules.
+
+## Prerequisites
+
+1. Several target machines that meet the following requirements:
+
+    - 4 or more machines
+
+        A standard TiKV cluster contains 6 machines. You can use 4 machines for test.
+
+    - CentOS 7.3 or later version of Linux operating system, x86_64 architecture (AMD64), ext4 filesystem
```

Comment:
Better to say " CentOS 7.3 (64 bit) or later with Python 2.7 installed)" (same as https://github.com/pingcap/docs/pull/482/files#r192772609)